lib/miquido_lints.yaml

```diff
@@ -0,0 +1,336 @@
+analyzer:
+  plugins:
+    - dart_code_metrics
+
+  # Explanation for this settings are described in Step 3 of this https://dash-overflow.net/articles/getting_started/
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+  
+  exclude:
+    - "**/*.g.dart"
+    - "**/*.freezed.dart"
+    - lib/generated/**
+    - generated_plugin_registrant.dart
+
+  language:
+    strict-raw-types: true
+    strict-casts: true
+    strict-inference: true
+
+  errors:
+    missing_required_param: error
+    missing_return: error
+    unawaited_futures: warning
+    unused_import: warning
+    prefer_const_constructors: info
+    prefer_const_constructors_in_immutables: info
+
+linter:
+  rules:
+    - always_declare_return_types
+    - always_put_control_body_on_new_line
+    - always_put_required_named_parameters_first
+    - always_require_non_null_named_parameters
+    # - always_specify_types - Incompatible with avoid_types_on_closure_parameters, which we prefer
+    - always_use_package_imports
+    - annotate_overrides
+    # - avoid_annotating_with_dynamic - Super problematic to fix with avoid_types_on_closure_parameters enabled
```

Comment:
We always want return types defined, so it shouldn't matter

Reply:
Shouldn't but I had difficult time with this one, maybe I am missing something